### ISSUE TITLE
Stats for each level

### ISSUE THREAD

**Issue by DanielVafidis** [samidhdk/Warcraft-Rumble-RESTful-API#2]
I'm new to participating in github so I'm not sure how to do but I would like to help maybe,
I think we should add stats for each of the 30 levels of the mins. maybe /units/stats/{unit_id}/{unit_level} ?

`
"stats": { 
    1 : {
        "Area Damage": 170,
        "Health": 3400,
        "DPS": 68,
        "Attack Speed": 2.5,
        "Speed": "Slow"
        }
    2 : {
         "Area Damage": 170,
        "Health": 3400,
        "DPS": 68,
        "Attack Speed": 2.5,
        "Speed": "Slow"
    )
}
`

Also Speed should be a trait, as we can have Fast as trait. also melee/AOE

**Comment by samidhdk** [comment]
Good morning Daniel, 

As for your first proposal, I'll have a look, but it does sound good. 

About the traits:
- Fast is a trait that represents that a unit has a speed of 4, the fastest one. Speed itself is only represented as a Stat in the game, so that's why I dont include it in the "search by trait".
- Melee and AoE are implemented in the API but not in the docs, I'll change that asap.

**Comment by DanielVafidis** [comment]
I thought fast was a trait as it can be found in leader slots (these are the traits right?)

**Comment by samidhdk** [comment]
Yeah Fast is the name of a Trait, and it means that it Speed Stat is "Fast".

Inside Stats there is an attribute "Speed", which can be any of the following: "Slow", "Med-Slow", "Medium", "Med-Fast", "Fast"

So, essentially a unit that has the trait "Fast" always has the "Speed Stat" as "Fast"

Also, it is not Leader exclusive: 

- Baron Rivendare has the fast trait
- Harpies have the fast trait aswell
